improver/wxcode/modal_code.py

```diff
@@ -733,6 +739,35 @@ def _set_blended_times(cube: Cube, result: Cube) -> None:
             )
             result.replace_coord(new_coord)
 
+    def _get_cloud_equivalents(
```

Comment:
Would the purpose of this function be clearer if it were called `get_dry_equivalents` or `get_dry_cloud_equivalents`?